developer-docs-site/docs/concepts/aptos-token.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Aptos Token"
+id: "aptos-token"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Aptos Token
+
+For digital assets, Aptos provides two Move modules:
+- The [`token.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/token.move) for the token standard. The token standard covers non-fungible (NFT), semi-fungible and **fungible non-decimal** token.
+- The [`coin.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/coin.move) for the coin standard. The coin standard specifies the **fungible decimal** token. 
```

Comment:
this isn't how I would differentiate.

token.move encapsulates collectibles and rich fungible and nonfungible assets
coin.move focuses on simple, typesafe, fungible assets